packages/x-components/src/x-modules/history-queries/components/__tests__/my-history.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { HistoryQuery } from '@empathyco/x-types';
+import { createLocalVue, mount, Wrapper, WrapperArray } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuex, { Store } from 'vuex';
+import { createHistoryQueries } from '../../../../__stubs__/index';
+import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
+import { getXComponentXModuleName, isXComponent } from '../../../../components/index';
+import { RootXStoreState } from '../../../../store/index';
+import { DeepPartial } from '../../../../utils/index';
+import { historyQueriesXModule } from '../../x-module';
+import MyHistory from '../my-history.vue';
+import { resetXHistoryQueriesStateWith } from './utils';
+
+function renderMyHistory({
+  template = '<MyHistory />',
+  historyQueries = []
+}: MyHistoryOptions = {}): MyHistoryAPI {
+  const localVue = createLocalVue();
+  localVue.use(Vuex);
+
+  const store = new Store<DeepPartial<RootXStoreState>>({});
+  installNewXPlugin({ store, initialXModules: [historyQueriesXModule] }, localVue);
+  resetXHistoryQueriesStateWith(store, { historyQueries });
+
+  const wrapper = mount(
+    {
+      template,
+      components: {
+        MyHistory
+      }
+    },
+    {
+      localVue,
+      store,
+      propsData: {
+        historyQueries
+      }
+    }
+  );
+  return {
+    wrapper: wrapper.findComponent(MyHistory),
+    search(query) {
+      wrapper.vm.$x.emit('UserAcceptedAQuery', query);
+      return wrapper.vm.$nextTick();
+    },
+    getListItems() {
+      return wrapper.findAll(getDataTestSelector('suggestion-item'));
+    },
+    findAllInWrapper(selector) {
+      return wrapper.findAll(getDataTestSelector(selector));
+    }
+  };
+}
+
+describe('testing MyHistory component', () => {
+  it('is an XComponent which has an XModule', () => {
+    const { wrapper } = renderMyHistory();
+    expect(isXComponent(wrapper.vm)).toEqual(true);
+    expect(getXComponentXModuleName(wrapper.vm)).toEqual('historyQueries');
+  });
+
+  it('does not render the component if history is empty', () => {
+    const { wrapper } = renderMyHistory();
+    expect(wrapper.html()).toEqual('');
+  });
+
+  it('renders the list of searched queries', async () => {
+    const { search, getListItems } = renderMyHistory();
+
+    await search('lego');
+    const suggestionsWrappers = getListItems();
+    expect(suggestionsWrappers.wrappers).toHaveLength(1);
+    expect(suggestionsWrappers.at(0).text()).toEqual('lego ✕');
+  });
+
+  describe('changing history query content', () => {
```

Comment:
You don't need to nest under a `describe` block if you are only going to add a single test